src/base/utils/net.cpp

```diff
@@ -38,5 +38,23 @@ namespace Utils
         {
             return !QHostAddress(ip).isNull();
         }
+
+        bool isLoopbackAddress(const QHostAddress &clientAddress)
+        {
+            return clientAddress == QHostAddress::LocalHost
+                    || clientAddress == QHostAddress::LocalHostIPv6
+                    || clientAddress == QHostAddress("::ffff:127.0.0.1");
+        }
+
+        bool isLANAddress(const QHostAddress &clientAddress)
+        {
+            return clientAddress.isInSubnet(QHostAddress::parseSubnet("10.0.0.0/8"))
```

Comment:
> isInSubnet(QLatin1String("10.0.0.0"), 8)
What's the purpose of using QLatin1String in this instance? It's not currently used for creating the ipv6 mapped 127.0.0.1 address.